bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/thing/charge-statistics-channel-types.xml

```diff
@@ -11,11 +11,19 @@
 		<item-type>Number:Energy</item-type>
 		<label>Energy Charged</label>
 		<description>Total energy charged in current month</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
 		<state pattern="%d %unit%" readOnly="true"/>
 	</channel-type>
 	<channel-type id="statistic-sessions-channel">
 		<item-type>Number</item-type>
 		<label>Charge Sessions</label>
 		<description>Number of charging sessions this month</description>
+		<tags>
+			<tag>Control</tag>
+			<tag>Channel</tag>
```

Comment:
I think this is not correct, but don't know what to use instead. Would a `Count` property make sense?

Reply:
Ditto `Status`

Reply:
No `Count` existing -- do you want to add it? Otherwise proposing plain `Status`..